apps/api/v1/lib/validations/team.ts

```diff
@@ -6,6 +6,9 @@ export const schemaTeamBaseBodyParams = Team.omit({ id: true, createdAt: true })
   hideBranding: true,
   metadata: true,
   pendingPayment: true,
+  isOrganization: true,
+  isPlatform: true,
+  smsLockState: true,
```

Comment:
Marks these newly added mandatory columns as partial to allow team creation without them as input (they're not relevant for API V1)